pacman/model/graphs/machine/__init__.py

```diff
@@ -15,11 +15,26 @@
 
 from .machine_edge import MachineEdge
 from .machine_fpga_vertex import MachineFPGAVertex
-from .machine_graph import MachineGraph
+from .machine_sdram_edge import SDRAMMachineEdge
 from .machine_spinnaker_link_vertex import MachineSpiNNakerLinkVertex
 from .machine_vertex import MachineVertex
 from .simple_machine_vertex import SimpleMachineVertex
+from .abstract_machine_edge_partition import AbstractMachineEdgePartition
+from .abstract_sdram_partition import AbstractSDRAMPartition
+from .constant_sdram_machine_partition import ConstantSDRAMMachinePartition
+from .destination_segmented_sdram_machine_partition import (
+    DestinationSegmentedSDRAMMachinePartition)
+from .fixed_route_edge_partition import FixedRouteEdgePartition
+from .multicast_edge_partition import MulticastEdgePartition
+from .source_segmented_sdram_machine_partition import (
+    SourceSegmentedSDRAMMachinePartition)
+from .machine_graph import MachineGraph
```

Comment:
Is there a reason why .machine_graph import has to be here and not where it was originally (in the middle of the other .machine_* imports)?

Reply:
fixed........maybe

Reply:
nope. there was a reason afterall. needs to be where it was.